packages/toolpad-core/src/appDom.ts

```diff
@@ -1184,3 +1187,8 @@ export function getPageTitle(node: PageNode): string {
 export function isCodePage(node: PageNode): boolean {
   return !!node.attributes.codeFile;
 }
+
+export function getPlan(dom: AppDom): ToolpadPlan | undefined {
```

Comment:
Why not make this apply the default?
```suggestion
export function getPlan(dom: AppDom): ToolpadPlan {
```